lessons/models.py

```diff
@@ -62,6 +62,18 @@ class Meta:
         ordering = ["word"]
         verbose_name_plural = "vocab words"
 
+    @property
+    def i18n_key(self):
+        return slugify(self.get_untranslated_field('word'))
+
+    @classmethod
+    def internationalizable_fields(cls):
+        return ['word', 'simpleDef', 'detailDef']
+
+    @property
+    def should_be_translated(self):
+        return any(lesson.should_be_translated for lesson in self.lesson_set.all()) and not self.mathy
```

Comment:
nit:
```suggestion
        return not self.mathy and any(lesson.should_be_translated for lesson in self.lesson_set.all())
```

saves us from an expensive query for all mathy values